e2e-pw/src/oss/poms/field-visibility/field-visibility.ts

```diff
@@ -63,4 +209,68 @@ class FieldVisibilityAsserter {
     private readonly fv: FieldVisibilityPom,
     private readonly sb: SidebarPom
   ) {}
+
+  async fieldVisibilityIconHasTooltip() {
+    await this.fv.fieldVisibilityBtn.hover();
+    await expect(this.fv.fieldVisibilityToggleTooltip).toBeVisible();
+  }
+
+  async assertAllFieldsSelected() {
```

Comment:
A generic approach could have the `selectionFields` as an argument
```suggestion
  async assertAllFieldsSelected(selectionFields: string[]) {
```

Reply:
good idea!